README.md

```diff
@@ -29,15 +29,32 @@ avoids breaking changes other than at major (x.0.0) releases. However,
 occasionally Azure API fixes require breaking updates within an individual
 package; these exceptions are noted in release changelogs.
 
+Because Azure's APIs are updated frequently, we release a **new major version
+at the end of each month**. For more details and background see [SDK Update
+Practices](https://github.com/Azure/azure-sdk-for-go/wiki/SDK-Update-Practices).
+
 To more reliably manage dependencies like the Azure SDK in your applications we
 recommend [golang/dep](https://github.com/golang/dep).
 
+## Other Azure Go Packages
+
+Azure provides several other packages for using services from Go, listed below.
+If a package you need isn't available please open an issue and let us know.
+
+| Service | Import Path/Repo |
+|---------|------------------|
+| Storage - Blobs | [github.com/Azure/azure-storage-blob-go](https://github.com/Azure/azure-storage-blob-go) |
+| Storage - Files | [github.com/Azure/azure-storage-file-go](https://github.com/Azure/azure-storage-file-go) |
+| Storage - Queues | [github.com/Azure/azure-storage-queue-go](https://github.com/Azure/azure-storage-queue-go) |
+| Event Hubs | [github.com/Azure/azure-event-hubs-go](https://github.com/Azure/azure-event-hubs-go) |
+| Application Insights | [github.com/Microsoft/ApplicationInsights-go](https://github.com/Microsoft/ApplicationInsights-go) |
+
 # Install and Use:
 
-### Install
+## Install
 
 ```sh
-$ go get -u github.com/Azure/azure-sdk-for-go/...
+$ go get -u github.com/Azure/azure-sdk-for-go
```

Comment:
This will fail to fetch any transitive dependencies. The most sure fire way to get everything you need is to include the `/...`.